doc/onboarding.md

```diff
@@ -100,18 +107,20 @@ onboarding session.
         (especially if it just has nits left).
   * Approving a change
     * Collaborators indicate that they have reviewed and approve of the
-        the changes in a pull request by commenting with `LGTM`, which stands
-        for "looks good to me".
-    * You have the power to `LGTM` another collaborator's (including TSC/CTC
-        members) work.
-    * You may not `LGTM` your own pull requests.
-    * You have the power to `LGTM` anyone else's pull requests.
+        the changes in a pull request using Github’s approval interface
+    * Some people like to comment `LGTM` (“Looks Good To Me”)
```

Comment:
I like to do this, mostly because it does look a lot nicer to me (and partly because I like to know if something breaks because of this – if something can’t handle UTF-8 in 2017, it’s useless). I’ve done this before in the docs, and so far nobody has complained.